spec/models/social_account_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe SocialAccount, type: :model do
+    before(:each) do
+        @social_account = FactoryBot.build(:social_account)
+    end
+
+    it "is creatable using a factory" do
+        expect(@social_account).to be_valid
+    end
+
+    it "is not valid without an social network" do
+        @social_account.social_network = ""
+        expect(@social_account).to_not be_valid
```

Comment:
(Hm, some specs in our solution are using `not_to`, others are using `to_not`. Should we agree on one solution for the whole repo? Maybe sharpen `RuboCop::Cop::RSpec::NotToNot` in our linter?)